osu.Framework/Platform/IWindow.cs

```diff
@@ -186,6 +186,24 @@ public interface IWindow : IDisposable
         /// </summary>
         Size ClientSize { get; }
 
+        /// <summary>
+        /// The minimum size of the window.
+        /// </summary>
+        /// <remarks>
+        /// To remove this constraint, set an empty <see cref="Size"/>: <c>Size.Empty</c>.
+        /// </remarks>
+        /// <exception cref="InvalidOperationException">Thrown when setting a negative size, or size greater than <see cref="MaxSize"/>.</exception>
+        Size MinSize { get; set; }
+
+        /// <summary>
+        /// The maximum size of the window.
+        /// </summary>
+        /// <remarks>
+        /// To remove this constraint, set a <see cref="Size"/> of maximum integer value: <c>new Size(int.MaxValue, int.MaxValue)</c>.
+        /// </remarks>
+        /// <exception cref="InvalidOperationException">Thrown when setting a negative or zero size, or size less than or equal to <see cref="MinSize"/>.</exception>
```

Comment:
```suggestion
        /// <exception cref="InvalidOperationException">Thrown when setting a negative or zero size, or a size less than <see cref="MinSize"/>.</exception>
```
It's just less than, right?

Reply:
Well, it can be less than or equal to, at which point it makes the window basically unresizable. Though that defeats the purpose of `Resizable`, do you think it should just throw?

Reply:
Imo it doesn't need to throw, and the code currently doesn't throw if it's equal to.

Reply:
Ah, nevermind, you're right.